validator/client/validator.go

```diff
@@ -176,6 +179,32 @@ func (v *validator) WaitForChainStart(ctx context.Context) error {
 	return nil
 }
 
+// Poll the beacon node every chainStartPollingInterval until the node
+// is available.
+func (v *validator) queryChainStart(ctx context.Context) (ethpb.BeaconNodeValidator_WaitForChainStartClient, error) {
+	stream, err := v.validatorClient.WaitForChainStart(ctx, &ptypes.Empty{})
+	if err == nil {
+		return stream, nil
+	}
+	log.WithError(err).Error("Could not fetch chain start from beacon node")
```

Comment:
Wouldn't we always be logging `Could not fetch chain start from beacon node` here?

Reply:
could you clarify. i changed the logging to warning wdyt?